scripts/india_census/primary_religion_data/preprocess.py

```diff
@@ -147,82 +215,112 @@ def stat_var_index_key(row, data_category_column=None):
     def _get_base_name(self, row):
         # To make the name meaningful add Religion to the
         # the name of the stat var
-        name = "Count_" + row["populationType"] + "_Religion"
+        name = "Count_" + row["populationType"]
         return name
 
+    def _get_base_constraints(self, row):
+        constraints = ""
+        return constraints
+
     def _get_stat_var_name(self, name):
-        return "dcid:indianCensus/{}".format(name)
+        return "dcid:{}".format(name)
 
     def _get_measured_property_name(self, name):
-        return "dcs:indianCensus/{}".format(name)
+        return "count"
```

Comment:
dcs:count

Reply:
Its handled as part of  `TEMPLATE_STAT_VAR` template. See `measuredProperty: dcs:{measuredProperty}`